Code style docs

Dansoftowner opened the pull request.

Dansoftowner committed:
:construction: ADD CODE_STYLE.md

Dansoftowner committed:
:construction: Update CODE_STYLE.md

Dansoftowner committed:
:memo: CODE_STYLE.md

feherszabolcs reviewed (changes requested):
remove "Kisebb fájlokat könnyebb megérteni (200 sor/fájl az ideális)", not possible in React :(

Dansoftowner committed:
:memo: UPDATE CODE_STYLE.md

KubikArmin committed:
:memo: Update CODE_STYLE.md

Nézd át, nem muszáj mindenhol az én verziómat írni. 
Az "Általános" részt írtam át jobban, mert felváltva voltak kijelentő és felszólító mondatok, ami nem tűnt konzekvensnek egy dokumentációban.

Dansoftowner committed:
:memo: Update CODE_STYLE.md
